stories/index.stories.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
```

Comment:
please add the story next to its component (like test files). no need a dedicated folder for these files